### PR TITLE
acrn-config: Support whiskylake i5 and whiskylake i7 board

### PR DESCRIPTION
**Pull request by liuwei987**
- refine mem_size_set function
- refine interrupt_storm function
- add support to generate launch script
- remove runC script from unnecessary launch script
- add config files for whl-ipc-i7 board
- add config files for whl-ipc-i5 board
- remove parser for console

    Tracked-On: #3854
    Signed-off-by: Wei Liu <weix.w.liu@intel.com>
    Acked-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: misc/acrn-config/launch_config/com.py [hunk 1]
```diff
@@ -276,32 +275,34 @@ def uos_launch(names, args, vmid, config):
     uos_type = names['uos_types'][vmid]
     launch_uos = launch_cfg_lib.undline_name(uos_type).lower()
 
+    # runc_container is clearlinux uos type, and apl-up2 dose not support for now
     if uos_type in ("CLEARLINUX", "ANDROID", "ALIOS") and not is_nuc_clr(names, vmid):
```
Comment (binbinwu1) [inline misc/acrn-config/launch_config/com.py]: does apl-up2 equals is_nuc_clr?

Comment (binbinwu1) [reply misc/acrn-config/launch_config/com.py]: Also a typo in the commit message "luanch" -> "launch"

Comment (liuwei987) [reply misc/acrn-config/launch_config/com.py]: apl-up2 does not equals is_nuc_clr, is_nuc_clr is the nuc board with clearlinux uos.
The typo will be corrected.

**Pull request merged**